src/stores/login.ts

```diff
@@ -0,0 +1,26 @@
+import { defineStore } from 'pinia';
+
+export const useLoginStore = defineStore('login', {
+  state: () => ({
+    user: {
+      email: '',
```

Comment:
Default value should be `null`, no (same for password key)?

Reply:
Ok, updated.
But the value needs to be passed into an input as a `string`, so I changed the getter to the "string" getter and sanitized the type there.